necst/procedures/observations/optical_pointing.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+from typing import Optional, Tuple
+import time
+
+from neclib import config
+from neclib.coordinates.observations import OpticalPointingSpec
+
+from .observation_base import Observation
+
+
+class OpticalPointing(Observation):
+    observation_type = "OpticalPointing"
+
+    def run(
+        self,
+        file: str,
+        magnitude: Tuple[float, float],
+        drive_test: bool = False,
+        obstime: Optional[datetime] = None,
+    ) -> None:
+        self.com.record("reduce", nth=60)
+        if obstime is None:
+            obsdatetime = datetime.now()
+        else:
+            obsdatetime = obstime
+        obsfloattime = obsdatetime.timestamp()
+        opt_pointing = OpticalPointingSpec(obsfloattime, "unix")
+        sorted_list = opt_pointing.sort(
+            catalog_file=file, magnitude=(float(magnitude[0]), float(magnitude[1]))
+        )
+        if obstime is None:
+            self.logger.info(
+                f"{len(sorted_list)} stars will be captured. Do you want to start?"
+            )
+            _input = input("(y/n) ")
+            if _input != "y":
+                self.logger.info("System ended.")
+                return None
+        else:
+            self.logger.info(f"{len(sorted_list)} stars will be captured.")
+            return None
+        self.logger.info("Starting Optical Pointing Observation.")
+        # self.logger.info(
+        #     f"Starting Optical Pointing Observation. Estimated observing time is {estimated_time} min."
```

Comment:
Flake8のCode Formatterのエラーです。まずこの行が文字数制限を超えているとのことです。このまま消してもいいのですが、将来的に観測完了予定時間の追加はしても良いと思いますのでTODOとしてください。
```suggestion
        # TODO: Add time estimator before observing.
```